### PR TITLE
Seed database with restaurant information

### PR DESCRIPTION
**Pull request by Sam-I-Am92**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+seedData.txt
```
Comment (cminho921) [inline .gitignore:2]: why do your seed data have a txt extension?

Comment (Sam-I-Am92) [reply .gitignore:2]: That's not the actual data. It's just pri's list copied from slack.

File: client/dist/bundle.js [hunk 2]
```diff
@@ -94,7 +94,7 @@
...
...WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(react__WEBPACK_IMPORTED_MODULE_0__);\n/* harmony import */ var react_dom__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! react-dom */ \"./node_modules/react-dom/index.js\");\n/* harmony import */ var react_dom__WEBPACK_IMPORTED_MODULE_1___default = /*#__PURE__*/__webpack_require__.n(react_dom__WEBPACK_IMPORTED_MODULE_1__);\n\n\nreact_dom__WEBPACK_IMPORTED_MODULE_1___default.a.render(react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(\"h1\", null, \"Goodbye, world!\"), document.getElementById('app'));//# sourceURL=[module]\n//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiLi9jbGllbnQvc3JjL2luZGV4LmpzeC5qcyIsInNvdXJjZXMiOlsid2VicGFjazovLy8uL2NsaWVudC9zcmMvaW5kZXguanN4P2QwMzAiXSwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0JztcbmltcG9ydCBSZWFjdERPTSBmcm9tICdyZWFjdC1kb20nO1xuXG5SZWFjdERPTS5yZW5kZXIoXG4gIDxoMT5Hb29kYnllLCB3b3JsZCE8L2gxPixcbiAgZG9jdW1lbnQuZ2V0RWxlbWVudEJ5SWQoJ2FwcCcpXG4pOyJdLCJtYXBwaW5ncyI6IkFBQUE7QUFBQTtBQUFBO0FBQUE7QUFBQTtBQUFBO0FBQ0E7QUFFQSIsInNvdXJjZVJvb3QiOiIifQ==\n//# sourceURL=webpack-internal:///./client/src/index.jsx\n");
```
Comment (cminho921) [inline client/dist/bundle.js:97]: you should add your bundle.js to gitignore

File: models/gallery.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+var Schema = mongoose.Schema;
+
+var schema = new Schema({
+  // object defining schema
+	restaurant_name: {type: String, required: true},
+  restaurant_id: {type: String, required: true},
+  // url: {
+  //   url1: String,
+  //   urln: String,
+  // },
```
Comment (cminho921) [inline models/gallery.js:11]: you need to clean up unnecessary lines commented out

File: seed/gallerySeeder.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+};
+
+function exit() {
+  mongoose.disconnect();
+};
+
+// new Gallery({
+//   restaurant_name: ,
+//   restaurant_id: 
+// }),
```
Comment (cminho921) [inline seed/gallerySeeder.js:51]: unnecessary comment lines

**Review by cminho921 — COMMENTED** [review]
Need to clean up the code, and add bundle.js to gitignore. you have to install enlint to your dependencies and change your code style according to the eslint airbnb style guide.

**Pull request closed without merging**